Problems/PoissonSinx/poisson_sinx_uniform.c

```diff
@@ -271,18 +268,18 @@ problem_init
 
     P4EST_FREE(u_analytic);
     P4EST_FREE(error);
-
-
+    
     // Compute and save norms
     d4est_norms_save(
       p4est,
       (const char * []){ "u", NULL },
       (double * []){ prob_vecs.u },
       (double * []){ NULL },
       (d4est_xyz_fcn_t[]){ poisson_sinx_analytic_solution },
-      &ctx,
-      (d4est_norms_fcn_t[]){ d4est_norms_fcn_L2, d4est_norms_fcn_Linfty, -1 },
-      (void * []){ &L2_norm_ctx, &Linfty_norm_ctx }
+      (void * []) { &ctx },
+      (const char * []){"L_2", "L_infty", NULL},
+      (d4est_norm_fcn_t[]){ &d4est_norms_L2, &d4est_norms_Linfty },
```

Comment:
I don't think this needs the & in front of d4est_norms_l2 and d4est_norms_Linfty, but maybe I am wrong.

Reply:
It works like this, I haven't tried removing the `&` 😄

Reply:
Okay then nvm :)